src/main/groovy/org/gradle/android/workarounds/RoomSchemaLocationWorkaround.groovy

```diff
@@ -54,13 +54,13 @@ import java.lang.reflect.Field
 class RoomSchemaLocationWorkaround implements Workaround {
     public static final String WORKAROUND_ENABLED_PROPERTY = "org.gradle.android.cache-fix.RoomSchemaLocationWorkaround.enabled"
     public static final String ROOM_SCHEMA_LOCATION = "room.schemaLocation"
-    private static final VersionNumber MINIMUM_KOTLIN_VERSION = VersionNumber.parse("1.4.32")
+    private static final VersionNumber MINIMUM_KOTLIN_VERSION = VersionNumber.parse("1.6.10")
```

Comment:
out of curiousity, why 1.6.10 and not 1.6.0?

Reply:
you're right, changed